File: Server/ServerProject/src/main/java/com/dani/objects/Target.java
package com.dani.objects;

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;

@JacksonXmlRootElement(localName = "targets")
public class Target {
    private Integer posX;
    private Integer posY;

    public Target() {
    }

    public Target(Integer posX, Integer posY) {
        this.posX = posX;
        this.posY = posY;
    }

    public Integer getPosX() {
        return posX;
    }

    public void setPosX(Integer posX) {
        this.posX = posX;
    }

    public Integer getPosY() {
        return posY;
    }

    public void setPosY(Integer posY) {
        this.posY = posY;
    }

    @Override
    public String toString() {
        return "{" +
                "posX=" + posX +
                ", posY=" + posY +
                '}';
    }
}

File: AppMovil/app/src/main/java/com/dani/appmovil/objectsWorld/Target.kt
package com.dani.appmovil.objectsWorld

import java.io.Serializable

data class Target (var posX:Integer?, var posY:Integer?):
    Serializable {
    constructor():this(Integer(0), Integer(0))
    override fun toString(): String {
        return "Target(posX='$posX', posY='$posY')"
    }
    fun clean() {
        posX = null
        posY = null
    }
}